### DISCUSSION TITLE
how can i have sshd run automatically as i open termux

### DISCUSSION THREAD

**Discussion by MMMMMoris** [termux/termux-packages#9586] [Q&A] [answered]
I feel upset every time  i input the address of my Termux on a PC terminal and try  sshconnect, however find out my Termux did not excute the order "sshd"... 
It's troublesome to input "sshd" on Termux each time i need to use it, how can i have i run automatically？ 
Since there is no such thing as "systemctl" on Termux...

**Comment by agnostic-apollo** [comment]
For now just run in `~/.bashrc`

**Reply by MMMMMoris** [reply]
could you please  give me an example ? i don't  get you somehow ...

**Reply by agnostic-apollo** [reply]
Search on internet.

https://unix.stackexchange.com/questions/129143/what-is-the-purpose-of-bashrc-and-how-does-it-work

**Comment by macerg** [comment, marked as answer]
In /data/data/com.termux/files/home/.termux/boot
#!/data/data/com.termux/files/usr/bin/sh
termux-wake-lock
sshd

This will put termux in 'non sleep' mode and run sshd on phone boot presuming you have termux:boot installed.

**Reply by MMMMMoris** [reply]
thanks a lot, I have this problem solved with the method you said, thanks!!

**Reply by macerg** [reply]
Happy to be of help.
